main.go

```diff
@@ -137,6 +140,12 @@ func coverage(coverpkg, pkg, covermode, cpu, parallel, timeout string, short, v
 	if v {
 		args = append(args, "-v")
 	}
+	if race {
+		if covermode != "" && covermode != "atomic" {
+			log.Fatalf("cannot use race flag and covermode=%s. See more detail on golang/go#12118.", covermode)
```

Comment:
@haya14busa 
> I think I should have handle the error from coverage() regardless this pull-request... sorry

Yeah. I think so, too. 
At first, I'm trying to return the `err`, but seems to not handled. So I used it while understanding `log.Fatalf` is bad way :P

> Anyway, I think I should refactor the code... additional test flag won't be changed , so we can move the code from coverage() to run().

Is that meaning is like this? (Sorry, I am sleep deprived lately, so my head doesn't work lol)

```diff
diff --git a/main.go b/main.go
index b56b738..a97cbc9 100644
--- a/main.go
+++ b/main.go
@@ -60,6 +60,10 @@ func run(coverprofile string, args []string, covermode, cpu, parallel, timeout s
 		usage()
 		return nil
 	}
+	if covermode != "" && covermode != "atomic" {
+		log.Fatalf("cannot use race flag and covermode=%s. See more detail on golang/go#12118.", covermode)
+	}
+
 	file, err := os.Create(coverprofile)
 	if err != nil {
 		return err
@@ -141,9 +145,6 @@ func coverage(coverpkg, pkg, covermode, cpu, parallel, timeout string, short, v
 		args = append(args, "-v")
 	}
 	if race {
-		if covermode != "" && covermode != "atomic" {
-			log.Fatalf("cannot use race flag and covermode=%s. See more detail on golang/go#12118.", covermode)
-		}
 		args = append(args, "-race")
 	}
 	cmd := exec.Command("go", args...)
```